### DISCUSSION TITLE
Agent Committee Meeting, today, Wednesday 10/12/2022

### DISCUSSION THREAD

**Discussion by lin-fred** [ArctosDB/arctos#6137] [Agent Committee]
Hey everyone,

Reminder the next Agent meeting is today, Wednesday the 12th at 1:30pm MTN time. You can see the [Arctos calendar](https://calendar.google.com/calendar/u/2?cid=amt2OWY4a2UwYXBxamJzOG52c3FybTRyODRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) for details and zoom link. Please contact me if you have issues using the zoom information.

Take a look at the agents project page https://github.com/orgs/ArctosDB/projects/16 under the Agent Committee Discussion/Verbatim Agents. Look over the issues, we will be going over them in order from top to bottom. Please add any that you want to discuss!

Agenda will be here:
https://docs.google.com/document/d/1p6z8s3_jsDelPxqxkXZ4ZqH8kzovS49I2gg_-HWVk68/edit#heading=h.1550euepfxv2

Lindsey

**Comment by Jegelewicz** [comment]
I'm sorry I will miss most of this meeting because I have to represent Arctos at the iDigbio webinar today. I'll try to show up for the last 30 minutes!
